docs/content/topics/scripting/guide.md

```diff
@@ -9,1401 +9,867 @@ aliases:
+  job2.primaryContainer.command = ["echo"];
+  job2.primaryContainer.arguments = ["My second job!"];
+  
+  let jobA = new Job("my-a-job", "debian", event);
+  jobA.primaryContainer.command = ["echo"];
+  jobA.primaryContainer.arguments = ["My A job!"];
 
-  test.run()
-})
-```
-[brigade-09.js](../../examples/brigade-09.js)
-
-This time around, we are going to run three tasks:
-
-- `cd /src/hello`: Change directories into the place where our source code is.
-- `yarn install`: Install the dependencies for our Node.js app. (Yarn is like pip, Maven,
-  Glide or CPAN, but for Node.js.)
-- `node index.js`: Run the `index.js` file inside of `/src/hello`.
-
-If we run the script, we'll see the usual output, and it will contain a line like this:
-
-```
-Creating pod test-app-1504064455281-master
-```
+  let jobB = new Job("my-b-job", "debian", event);
+  jobB.primaryContainer.command = ["echo"];
+  jobB.primaryContainer.arguments = ["My B job!"];
 
-If we use `kubectl` to check out the log for our `test-app` container, we'll see something
-like this:
+  await Job.concurrent(
+    Job.sequence(job1, job2),
+    Job.sequence(jobA, jobB)
+  ).run();
+});
 
-```console
-$ kubectl logs test-app-1504064455281-master
-yarn install v0.27.5
-info No lockfile found.
-[1/4] Resolving packages...
-[2/4] Fetching packages...
-[3/4] Linking dependencies...
-[4/4] Building fresh packages...
-success Saved lockfile.
-Done in 0.14s.
-hello world
+events.process();
 ```
+[05-groups](https://github.com/brigadecore/brigade/tree/v2/examples/05-groups)
 
-That is the output of our three tasks.
+There are two things to notice in the example above:
 
-But what exactly is this build doing? Why are we doing a `cd /src/hello`? And where did
-that directory even come from?
+1. Both the `concurrent` and `sequence` methods exist on the `Job` object.
+2. The return type for both methods is a generic "runnable", i.e. an object
+  that `run()` can then be called on, just like a standalone job instance.
 
-Here's what is happening: Because our project has a Git repository associated with it,
-Brigade is automatically getting us a copy of that project and attaching that copy to each
-`Job` that we run.
+Here's a breakdown of each method:
 
-When a repository is checked out, it is stored by default in `/src`. So it is as if we
-started every job by doing a `git clone https://github.com/brigadecore/empty-testbed.git /src`.
-That means we can start our job knowing that we already have access to everything in our
-Git project.
+- `Job.sequence()` takes an array of runnables (e.g. a job or a group of
+  jobs) and runs them _in sequence_. A new runnable is started only when the
+  previous one completes. The sequence completes when the last runnable has
+  completed (or when any runnable fails).
+- `Job.concurrent()` takes an array of runnables and runs them all
+  _in parallel_. When run, all runnables are started simultaneously (subject to
```

Comment:
Nit: There's a nuanced difference between "in parallel" and "concurrently." I'll refrain from waxing philosophical, but "concurrently" is the more general of the two terms, so I think we should prefer to use that when we can.